.github/workflows/zxcron-extended-test-suite.yaml

```diff
@@ -36,29 +41,37 @@ jobs:
     name: Fetch XTS Candidate Tag
     runs-on: network-node-linux-medium
     outputs:
-      xts_tag_exists: ${{ steps.check_tags_exist.outputs.xts_tag_exists }}
-      xts_tag_commit: ${{ steps.check_tags_exist.outputs.xts_tag_commit }}
+      xts_tag_exists: ${{ steps.check-tags-exist.outputs.xts_tag_exists }}
```

Comment:
Is xts_tag_exists able to be xts-tag-exists too, for consistency with check-tags-exist?

Reply:
Unfortunately not. `xts_tag_exists` is a bash variable and therefore cannot contain the `-` character. `check-tags-exist` is a job|step id and those have consistently used the `-` instead of the `_` in our workflows.